release/nightly_tests/dataset/multi_node_train_benchmark.py

```diff
@@ -102,38 +132,46 @@ def train_loop_per_worker():
             num_rows = 0
             start_t = time.time()
             for batch in it.iter_batches(
-                batch_size=args.batch_size, prefetch_batches=10
+                batch_size=args.batch_size,
+                local_shuffle_buffer_size=args.local_shuffle_buffer_size,
+                prefetch_batches=10,
             ):
-                num_rows += len(batch["image"])
+                num_rows += args.batch_size
             end_t = time.time()
             # Record throughput per epoch.
             epoch_tput = num_rows / (end_t - start_t)
             session.report({"tput": epoch_tput, "epoch": i})
 
     # 3) Train TorchTrainer on processed data
+    options = DataConfig.default_ingest_options()
+    options.preserve_order = args.preserve_order
+
     torch_trainer = TorchTrainer(
         train_loop_per_worker,
-        scaling_config=ScalingConfig(num_workers=args.num_workers),
         datasets={"train": ray_dataset},
+        scaling_config=ScalingConfig(num_workers=args.num_workers),
+        dataset_config=ray.train.DataConfig(
+            execution_options=options,
+        ),
     )
     result = torch_trainer.fit()
 
     # Report the throughput of the last training epoch.
-    metrics["ray.TorchTrainer.fit"] = list(result.metrics_dataframe["tput"])[-1]
+    metrics["ray_TorchTrainer_fit"] = list(result.metrics_dataframe["tput"])[-1]
```

Comment:
how this `metrics_dataframe` being aggregated across training workers? is this the sum up of individual `tput` from each worker?